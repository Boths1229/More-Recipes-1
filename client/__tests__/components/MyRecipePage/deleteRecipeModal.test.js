import React from 'react';
import { shallow } from 'enzyme';
import DeleteRecipeModal from '../../../components/MyRecipePage/deleteRecipeModal';

describe('DeleteRecipeModal Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<DeleteRecipeModal />);
    expect(wrapper).toMatchSnapshot();
    expect(wrapper.find('button').length).toBe(3);
    expect(wrapper.find('button').at(1).text()).toBe('Delete');
  });
});
