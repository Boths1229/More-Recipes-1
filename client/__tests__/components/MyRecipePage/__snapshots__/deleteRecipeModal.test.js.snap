// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeleteRecipeModal Component should render correctly 1`] = `
<div
  aria-hidden="true"
  aria-labelledby="deleteModalLabel"
  className="modal fade"
  data-backdrop="static"
  id="deleteModal"
  role="dialog"
  style={
    Object {
      "display": "none",
    }
  }
  tabIndex="-1"
>
  <div
    className="modal-dialog"
    role="document"
  >
    <div
      className="modal-content"
    >
      <div
        className="modal-header black"
      >
        <h5
          className="modal-title black"
          id="deleteModalLabel"
        >
          Delete recipe
        </h5>
        <button
          aria-label="Close"
          className="close"
          data-dismiss="modal"
          type="button"
        >
          <span
            aria-hidden="true"
          >
            Ã—
          </span>
        </button>
      </div>
      <div
        className="modal-body"
      >
        This action cannot be reversed, proceed with caution.
      </div>
      <div
        className="modal-footer"
      >
        <button
          className="btn btn-danger"
          id="deleteButton"
          type="button"
        >
          Delete
        </button>
        <button
          className="btn btn-secondary"
          data-dismiss="modal"
          type="button"
        >
          Close
        </button>
      </div>
    </div>
  </div>
</div>
`;
